docs/_docs/languages/graphql.md

```diff
@@ -0,0 +1,67 @@
+---
+pageid: language-graphql
+title: GraphQL
+layout: docs
+permalink: /docs/languages/graphql/
+---
+
+Nuclide has built-in support for [GraphQL](http://graphql.org/) using the [GraphQL Language Service](https://github.com/graphql/graphql-language-service).
+
+* TOC
+{:toc}
+
+## Installing GraphQL
+
+Several [server libraries](http://graphql.org/code/) are provided for GraphQL implementation in a wide range of languages.
+
+Opening a `.graphql` file in Nuclide will trigger the GraphQL support.
+
+## Features
+
+GraphQL's integration into Nuclide provides you with productivity features such as:
+
+- [Autocomplete](#features__autocomplete)
+- [Go to Definition](#features__go-to-definition)
+- [Outline View](#features__outline-view)
+- [Context View](#features__context-view)
+- [Code Diagnostics](#features__code-diagnostics)
+
+### Autocomplete
+
+Nuclide has access to the schema type information in your project, so autocomplete just works.
```

Comment:
Nuclide has access to the schema **with the GraphQL configuration file (`.graphqlrc`)** defined. I'm not sure where to put this information - still wanted to point this out.